source/transactions/tests/write-concern.yml

```diff
@@ -242,3 +247,401 @@ tests:
         result:
           # Client-side error.
           errorContains: transactions do not support unacknowledged write concern
+
+  - description: unacknowledged write concern coll insertOne
+
+    operations:
+      - *startTransaction
+      - name: insertOne
+        object: collection
+        collectionOptions:
+          writeConcern:
+            w: 0
```

Comment:
Would it make sense to use [Extend Inline](https://blog.daemonl.com/2016/02/yaml.html) to reduce some duplication here?

```yaml
operations:
  - *startTransaction
  - name: insertOne
    <<: &collection_w0
      object: collection
      collectionOptions:
        writeConcern: { w: 0 }
    arguments: ...
```

And later you can just merge in `*collection_w0`:

```yaml
operations:
  - *startTransaction
  - name: insertOne
    <<: *collection_w0
    arguments: ...
```

Reply:
Perfect, done.